front/pages/api/w/[wId]/webhook_sources/views/[viewId]/index.ts

```diff
@@ -73,6 +75,17 @@ async function handler(
         });
       }
 
+      // Validate that this is a system view
+      if (webhookSourceView.space.kind !== "system") {
+        return apiError(req, res, {
+          status_code: 400,
+          api_error: {
+            type: "invalid_request_error",
+            message: "Updates can only be performed on system views.",
+          },
+        });
+      }
```

Comment:
I think you can update a webhook source view name. The difference is that it does not propagate to other views... No ?

Reply:
Ok my bad !